overops-functions/src/main/java/com/takipi/udf/jira/JiraIntegrationFunction.java

```diff
@@ -62,6 +63,29 @@ static JiraIntegrationInput getJiraIntegrationInput(String rawInput) {
 			throw new IllegalArgumentException("'jiraPassword' is required");
 		}
 
+		// validate credentials by logging in
+		JiraRestClientFactory factory = new AsynchronousJiraRestClientFactory();
```

Comment:
We should probably only validate the credentials here when were coming from `validateInput` and not when coming from `execute`, as `execute` will also create a client and attempt to work with it so no reason to do it twice :)